tests/test_run.py

```diff
@@ -71,7 +71,7 @@ def test_watch_only(
     cli_result_list = from_table(cli_result.output)
     assert "Failed" in cli_result_list[2]
     assert "SELECT 'asd' AS externalId 'asd' AS name" == " ".join(cli_result_list[4])
-    assert "mismatched input ''asd'' expecting {<EOF>, ';'}(line 1, pos 27)" == " ".join(cli_result_list[6])
+    assert "Syntax error at or near ''asd''(line 1, pos 27)" == " ".join(cli_result_list[6])
```

Comment:
is this an error msg from the api? relaying on server side strings can make these tests flaky... any chance of testing for an http code, a status, of something with lower chance of changing?

Reply:
It is not a big deal, the error returned from failing transformation changed for some reason. I check if we propagate the error nicely, so I think worst case we change it again in cli prs.

Reply:
this is not sdk, so it doesn't block anyone else.